packages/plugins/src/SidebarPlugin.ts

```diff
@@ -164,15 +152,29 @@ const SidebarPlugin: PluginType<SidebarPluginPage, SidebarPluginOptions, Sidebar
         deep: sidebarRootLevel
       });
 
+      //Map into Sidebar Groups and sort children according to priority, priority = 1 has the highest priority
+      function sortSidebarGroups(sidebarData) {
+        const sortedGroupedPages = sidebarData.map(page => {
+          if (page.childNodes) {
+            const sortedPages = page.childNodes.sort((a, b) => a.priority - b.priority);
```

Comment:
looks like we might be changing the way that priority works. Consider that this means all the existing pages will need changing, so this becomes a breaking change and something which need communication as well as codemods.

Instead we could create an additive change and deprecate the old
priority. JP used to talk about creating a ‘sidebar-> sortLabel’ instead of using a numeric value

The value here is that it would be less brittle and more flexible as you would not need to change the other numbers if new entries are added.

One to discuss with wider team, get someone’s else’s view and see what the consensus is.

ie. folks tend to add ascending integers for priority when all they want to do is set the priority of one label

ie.. Say you want to order 3 labels, instead of priority 1,2.3 you could just add one ‘sortLabel’ which ensures that the label appears in the right position